db/migrate/20210915145551_create_alerts.rb

```diff
@@ -0,0 +1,18 @@
+class CreateAlerts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :alerts do |t|
+      t.string :name
+      t.string :metric_name, null: false
+      t.integer :threshold, null: false
+      t.string :emails, array: true, default: [], null: false
+      t.integer :frequency, null: false
+      t.datetime :last_sent_date
+      t.boolean :active
```

Comment:
same, we can't have nullable boolean columns https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem